### DISCUSSION TITLE
Java version in the Editor

### DISCUSSION THREAD

**Discussion by aghasemi** [jbangdev/jbang#1281] [Ideas] [answered]
Hi,
I just realised that when editing a JBang script with `jbang edit`, the Java version that is applied in the editor is that of the system (or the editor itself, more precisely), not the version specified with `//JAVA `directive. This can cause inconsistency when running a script in the editor and then using the `jbang run` command.

Is it a know issue? Is there any solution around it? I can understand it is a very difficult situation. 

Many thanks.

**Comment by maxandersen** [comment, marked as answer]
That's a bug.

**Reply by maxandersen** [reply]
@aghasemi can you open issue with info on what editor/IDE you see it ?

note: we probably can't handle all cases.
